executor/ddl_test.go

```diff
@@ -273,11 +273,24 @@ func (s *testSuite6) TestCreateView(c *C) {
 	tk.MustExec("create definer='root'@'localhost' view v_nested as select * from test_v_nested")
 	tk.MustExec("create definer='root'@'localhost' view v_nested2 as select * from v_nested")
 	_, err = tk.Exec("create or replace definer='root'@'localhost' view v_nested as select * from v_nested2")
-	c.Assert(terror.ErrorEqual(err, plannercore.ErrNoSuchTable), IsTrue)
+	c.Assert(terror.ErrorEqual(err, plannercore.ErrViewRecursive), IsTrue)
```

Comment:
It seems that this error message is not compatible with MySQL

Reply:
fixed